components/Card/card.css

```diff
@@ -0,0 +1,78 @@
+@import "/docs.css";
+@import "components\Button\button.css";
+/* @import "/components/Utilities/utilities.css"; */
+
+.card{
+    display: flex;
+    flex-direction: column;
+    max-width:min-content;
+    border-radius: 10px;
+    position:relative;
+    margin: 0.5rem;
+    background-color: var(--white-color);
+}
+
+.card-horizontal{
+    display: flex;
+    flex-direction: row;
```

Comment:
flex-direction is row by default, so no need to add it. It will reduce your code size if you are doing the same thing in multiple places.

Reply:
Removed.